src/fable-library/Seq.ts

```diff
@@ -81,7 +85,11 @@ function makeSeq<T>(f: () => Iterator<T>): Iterable<T> {
 }
 
 export function ofArray<T>(xs: ArrayLike<T>) {
-  return delay(() => unfold((i) => i != null && i < xs.length ? [xs[i], i + 1] : undefined, 0));
+  if (Array.isArray(xs)) {
```

Comment:
`Array.isArray` doesn't work with typed arrays (I guess this will be less important in Fable 3 with non-typed arrays as default, though we still compile FCS with typed arrays). There's an `isArray` function in Util.ts to test both cases.